roles/rds-apb-openshift/tasks/main.yml

```diff
@@ -49,3 +52,25 @@
       DB_PASSWORD: "{{ db_password }}"
       DB_NAME: "{{ db_name }}"
       DB_TYPE: postgres
+  when: state=="present"
+
+- name: delete rds
+  rds:
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    command: delete
```

Comment:
In the same vein as `k8s_v1_service`, instead of using `when` declarations to skip these actions, add the command as a  var in the provision/deprovision playbooks. For example,

```
+  vars:
+    state: 'present'
+    command: 'delete'
```